packages/migrations/CHANGELOG.json

```diff
@@ -5,6 +5,14 @@
             {
                 "note": "Upgrade the bignumber.js to v8.0.2",
                 "pr": 1517
+            },
+            {
+                "note": "Removed `owner` in Migrations. `txDefaults` is now a non-Partial type",
```

Comment:
> `txDefaults` is now a non-Partial type

nit: It's not very clear how this affects the API for callers. Should we just say something like "The `from` field is now required in the `txDefaults` argument for `runMigrationsAsync`"? That's what I would ultimately want to know if I were a developer using the `migrations` package as a library.